### PR TITLE
fix(container): fix node 18 bootstrap version 6.4.2

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
ref: MANAGER-12087

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-12087
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by chipp972 — DISMISSED** [review]

File: .husky/commit-msg [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn commitlint --edit $1
+npx commitlint --edit $1
```
Comment (anooparveti) [inline .husky/commit-msg:4]: **question:** Why switch from `yarn` to `npx`?

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged
+npx lint-staged
```
Comment (anooparveti) [inline .husky/pre-commit:4]: **question:** Why switch from `yarn` to `npx`?

File: .husky/commit-msg (hunk 1 above)
Comment (chipp972) [reply .husky/commit-msg:4]: https://github.com/typicode/husky/issues/1246

This is the fix for the available version of node that we have on the self service

**Review by anooparveti — DISMISSED** [review]

**Review by chipp972 — DISMISSED** [review]

File: .husky/pre-commit (hunk 2 above)
Comment (aboungnaseng-ovhcloud) [reply .husky/pre-commit]: I made a request to support team to have node v18.16.1 in the self service, to allow usage of 'yarn' and not 'npx' in the command husky

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Pull request merged**